doc/Writerside/topics/new_outcome_flow.md

```
@@ -0,0 +1,86 @@
+- В открывшемся дереве, отметьте реквизиты объекта которые необходимо отслеживать(В приведенном примере при изменении `Демо реквизит строка` регистрация нового сообщения не произойдет)
+   ![attrs_tree.png](attrs_tree.png) { style="block" }
+
+- Убедитесь, что ваши изменения применены. Должна появиться соответствующая декорация
+  ![decoration.png](decoration.png){ style="block" }
+
+## Заполнение подписчиков
+
+Выберите подписчиков, которым будет доставлено данное сообщение.
+
+Возможно перед тем, как выбрать подписчиков, их потребуется создать. Для создания подписчиков, прочитайте [эту инструкцию](subscribers.md).
+## Вместо заключения.
```

Markdownlint

Expected: 1; Actual: 0; Above
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)